utilities/src/errors.rs

```diff
@@ -87,6 +134,17 @@ pub trait PrettyUnwrap {
     fn pretty_unwrap(self) -> Self::Inner;
 }
 
+/// Set the global panic hook for process. Should be called exactly once.
+pub fn set_panic_hook() {
+    std::panic::set_hook(Box::new(|err| {
+        let msg = err.to_string();
+
+        println!("Hook: {msg}");
```

Comment:
Will this always print when an error is hit?

We had initially hidden all finalize panics/errors since it caused concerns and alerted node operators, despite expected VM behavior

Reply:
This was a leftover debug message. It is removed now.